omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_4/ServerLogController2_4Test.java

```diff
@@ -24,8 +24,8 @@
 import org.openmrs.module.webservices.rest.SimpleObject;
 import org.openmrs.module.webservices.rest.web.MockServerLogActionWrapper;
 import org.openmrs.module.webservices.rest.web.api.RestService;
-import org.openmrs.module.webservices.rest.web.v1_0.controller.jupiter.MainResourceControllerTest;
```

Comment:
Is this change part of the pull request?

Reply:
I am not sure what caused this but I had mistakenly moved that class to another package. In the latest push, I restored it and then pulled the latest changes on my master branch and rebased RESTWS-764.